io/disk/vfc-tests/vfc_dlpar.py

```diff
@@ -52,6 +52,7 @@ def setUp(self):
         self.hmc_pwd = self.params.get("hmc_pwd", '*', default=None)
         self.hmc_username = self.params.get("hmc_username", '*', default=None)
         self.count = self.params.get("count", default=1)
+        self.skip_drc = self.params.get("skip_drc_name", default=None)
```

Comment:
@Naresh-ibm Skipping by drc is the correct approach here.

But, it would be preferred to find that boot disk drc without user intervention.